src/services/test/goodreads-service.test.js

```diff
@@ -11,6 +11,14 @@ describe('search', () => {
     fetchSpy.mockClear();
   });
 
+  test('Call to api with book id to get the series the book is in', () => {
+    goodreadsService.getAllSeriesABookisIn(1);
+    expect(global.fetch).toHaveBeenCalledTimes(1);
+    expect(global.fetch).toHaveBeenCalledWith(
+      'https://www.goodreads.com/work/1/series?format=xml&key=key',
+    );
+  });
+
```

Comment:
I miss a test where you test that the response is the array in `GoodreadsResponse.series_works[0].series_work`